examples/reminders/lib/base/models/reminder/reminder_realm_model.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:realm/realm.dart';
+
+part 'reminder_realm_model.realm.dart';
+
+@RealmModel()
+class _ReminderRealmModel {
```

Comment:
Once again any realm related files should not be part of the businesses layer. They belongs to the data layer